src/igvfd/types/file_set.py

```diff
@@ -37,14 +37,18 @@ class FileSet(Item):
         'control_for': ('FileSet', 'control_file_sets')
     }
     embedded_with_frame = [
-        Path('award', include=['@id', 'component']),
+        Path('award', include=['@id', 'component', 'contact_pi']),
+        Path('award.contact_pi', include=['@id', 'title']),
```

Comment:
why not combine both into a single line: 
Path('award.contact_pi', include=['@id', 'title', 'component', 'contact_pi'])

Reply:
I think the way I have done it is more explicit and readable.